lms/src/ots/private.rs

```diff
@@ -107,7 +107,7 @@ impl<Mode: LmsOtsMode> RandomizedSignerMut<Signature<Mode>> for SigningKey<Mode>
 
         // Generate the message randomizer C
         let mut c = <Output<Mode::Hasher>>::default();
-        rng.fill_bytes(&mut c);
+        rng.try_fill_bytes(&mut c).map_err(|_| Error::new())?;
```

Comment:
We kind of eat the error here. This might be a bad idea.

I wish we could do Error::from_source instead, but that would require the RandomizedSignerMut to change its signature to something like:
```
fn try_sign_with_rng<E: core:error::Error, R: TryCryptoRng<Error = E>>(
        &mut self,
        rng: &mut R,
        msg: &[u8],
    ) -> Result<Signature<Mode>, Error> {
``
Not sure.

Reply:
We can potentially consider something like that